pkg/operator/controller/ingress/deployment.go

```diff
@@ -470,6 +484,22 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 	}
 	env = append(env, corev1.EnvVar{Name: RouterHAProxyThreadsEnvName, Value: strconv.Itoa(threads)})
 
+	if len(ci.Spec.TuningOptions.ClientTimeout) > 0 {
```

Comment:
Please can we extend the existing unig test case to assert these values are present when set on the spec.

Reply:
Will do

Reply:
Done :+1: